src/main/java/org/mskcc/cbio/oncokb/domain/UserStatusChecks.java

```diff
@@ -0,0 +1,47 @@
+package org.mskcc.cbio.oncokb.domain;
+
+import org.mskcc.cbio.oncokb.service.SlackService;
+import org.mskcc.cbio.oncokb.service.UserService;
+import org.mskcc.cbio.oncokb.service.dto.UserDTO;
+
+/**
+ * Created by Benjamin Xu on 7/13/21.
+ */
+public class UserStatusChecks {
+    UserDTO userDTO;
+
+    boolean trialAccountActivated;
+    boolean trialAccountInitiated;
+    boolean academicClarificationEmailSent;
+    boolean embargoEmailSent;
+
+    public UserStatusChecks(UserDTO userDTO, UserService userService, SlackService slackService) {
```

Comment:
We should not pass the service layout objects directly but the actual value to initialize the UserStatusChecks